README.md

```diff
@@ -1,2 +1,118 @@
-# juttle-engine
-juttle execution engine
+# Juttle Engine
+
+[![Build Status](https://travis-ci.org/juttle/juttle-engine.svg?branch=master)](https://travis-ci.org/juttle/juttle-engine)
```

Comment:
just enabled juttle-engine in travis only to realize that there isn't a `.travis.yml`. remove the badge for now?